src/rawtoaces_util/image_converter.cpp

```diff
@@ -869,7 +895,15 @@ void ImageConverter::init_parser( OIIO::ArgParse &arg_parser )
 
 bool ImageConverter::parse_parameters( const OIIO::ArgParse &arg_parser )
 {
-    settings.database_directories = database_paths();
+    std::string data_dir = arg_parser["data-dir"].get();
+    if ( data_dir.size() )
+    {
+        settings.database_directories = database_paths( data_dir );
+    }
+    else
+    {
+        settings.database_directories = database_paths();
+    }
```

Comment:
there are unit tests now for `list-cameras` and `list-illuminants`. They are the only but maybe the easest way you can test functionality above. Tests currently setup test-dit location via env. You could have one more test for cameras with same exact logic, but setup path with --data-dir and it would test new branch above.